### PR TITLE
ViewCreator3d default lighting update

### PR DESCRIPTION
**Pull request by Ellord207**
This change updates the default view created by the ViewCreator3d with brighter lighting.  A flag was also added to the ViewCreatorOptions that would allow users to op-out of the display styling changes.

### CODE REVIEW

File: core/frontend/src/ViewCreator3d.ts [hunk 1]
```diff
@@ -38,6 +38,8 @@ export interface ViewCreator3dOptions {
   useSeedView?: boolean;
   /** Aspect ratio of [[Viewport]]. Required to fit contents of the model(s) in the initial state of the view. */
   vpAspect?: number;
+  /** Use the old, dimmer stylings when generating view. */
+  useV2Stylings?: boolean;
```
Comment (pmconne) [inline core/frontend/src/ViewCreator3d.ts:42]: I think this should be called `useDefaultLighting` and documentation should clarify what it does if true vs what it does if false/undefined.

File: common/api/summary/core-common.exports.csv [hunk 2]
```diff
@@ -153,6 +153,7 @@ internal;DbResponseKind
 internal;DbResponseStatus
 beta;DbRuntimeStats
 internal;DecorationGeometryProps
+beta;DefaultSupportedTypes
```
Comment (pmconne) [inline common/api/summary/core-common.exports.csv:156]: This file shouldn't have been modified.

**Review by pmconne — CHANGES_REQUESTED** [review]
Are you intentionally not enabling ambient occlusion?

I would like to see a test that confirms the new option behaves as intended when true, false, and undefined.

**Comment by Ellord207** [comment]
Due to authentication issues.  I'm going to delete this branch and recreated it.  I'll try and move the comments over to the new PR assuming it doesn't re-open this one.

**Pull request closed without merging**